companies/fymio-de.json

```diff
@@ -0,0 +1,33 @@
+{
+    "slug": "fymio-de",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "fymio",
```

Comment:
From the privacy policy, the actual controller here is 'TeamBank AG'. You can add 'fymio' in the `runs` list.

Reply:
That's how I wrote it first.
However, I then saw that the TeamBank AG also has a [data protection](https://www.teambank.de/datenschutz/) section on their own [website](https://www.teambank.de/) with contact information that differs from the [information on fymio.de](https://www.fymio.de/datenschutz/) (`datenschutz@teambank.de` vs `service@www.fymio.de`)
How should these cases be handled?

Reply:
Hm, I see.

In this case, I would probably assume that there is a single DPO for both the company and Fymio (we prefer 'proper' emails anyway) and have a single record that lists `TeamBank AG` with `datenschutz@teambank.de` and Fymio in the `runs` list.  
I think that's a reasonable assumption to make. Worst case scenario, if there really are two DPOs, one will have to forward requests to the other. In that case, we would hopefully be notified by a user at some point and could change the record.

There are however sometimes (though rarely) cases where a single company has multiple DPOs for different services (like having `dpo@domain1.com` and `dpo@otherdomain.net` with different phone numbers etc.) In these cases, it would make sense to have two records.  
Then, we would essentially have two ways of creating these two records. Either:

```json
{
  "name": "Some Company Ltd.",
  "runs": [ "Domain One" ]
}
```

```json
{
  "name": "Some Company Ltd.",
  "runs": [ "Other Domain" ]
}
```

or alternatively:

```json
{
  "name": "Domain One",
  "address": "c/o Some Company Ltd.\nSome Street…"
}
```

```json
{
  "name": "Other Domain",
  "address": "c/o Some Company Ltd.\nSome Street…"
}
```

We would choose either of them depending on the particular case.

Reply:
I hope that's somewhat understandable. :D